client/dashboard/utils/purchase.ts

```diff
@@ -383,6 +383,10 @@ export function isJetpackSearch( product: ObjectWithProductSlug ): boolean {
 	return product.product_slug ? JETPACK_SEARCH_PRODUCTS.includes( product.product_slug ) : false;
 }
 
+export function isWpcomPlan( purchase: Purchase ): boolean {
```

Comment:
Super nit: I think it's better to rename this to `isDotcomPlan()`, because the constant for the plans itself is called `DotcomPlans`.